contracts/hackatom/schema/state.json

```diff
@@ -9,22 +9,19 @@
   ],
   "properties": {
     "beneficiary": {
-      "$ref": "#/definitions/CanonicalAddr"
+      "$ref": "#/definitions/Addr"
```

Comment:
Before this PR, state and messages types where in the main contract file. It did not look as if someone every thought about the difference here.

Reply:
> Wondering why I generated the schema for state anyway?

Yup, my fingers on auto-pilot when figuring out this JsonSchema stuff.
This contract dates back to June 2019 and I am sure it could use some refactors and best practices.